site/content/docs/5.0/migration.md

```diff
@@ -127,6 +126,7 @@ Breakpoints specific variants are consequently renamed too (eg. `.text-md-start`
   - Renamed `.font-style-*` utilities as `.fst-*` for brevity and consistency.
 - Added `.d-grid` to display utilities
 - Added new `gap` utilities (`.gap`) for CSS Grid layouts
+ - Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```

Comment:
Unwanted whitespace:

```suggestion
- Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```

Reply:
Oh and you still miss the link to #31687 :)

Reply:
This is my first time contributing so I didn't quite get it the first time where to put the link. I just made the changes please check if it's still not right.  😅

Reply:
Don't worry, you did well :+1:

Reply:
Thanks !